include/EVT/dev/LCD.hpp

```diff
@@ -0,0 +1,70 @@
+#ifndef _EVT_LCD_
+#define _EVT_LCD_
+
+#include <EVT/io/GPIO.hpp>
+#include <EVT/io/SPI.hpp>
+#include <EVT/io/UART.hpp>
+#include <EVT/io/pin.hpp>
+#include <EVT/utils/time.hpp>
+#include <stdint.h>
+
+#define ADCSELECT 0xA0
+#define DISPLAYOFF 0xAE
+#define COMDIRSCAN 0xC8
+#define LCDBIASET 0xA2
+#define POWERCONTROLSET 0x2F
+#define RESRATIOSET 0x26
+#define ELECTRONICVOLCOMMAND 0x81
+#define ELECTRONICVOLVALUE 0x11
+#define DISPLAYON 0xAF
+
+namespace EVT::core::DEV {
+/*
+* This class represents the structure to command a GLCD with 
+* a ST7565 controller.
+*/
+class LCD {
+
+public:
+    /// bit map to display the LCD's current state
+    unsigned char* bitMap;
+
+    ///
+    /*
+    * Constructor for the LCD class
+    *
+    * @param[in]
+    *
+    */
+    LCD(EVT::core::IO::GPIO& regSelect, EVT::core::IO::GPIO& reset, EVT::core::IO::SPI& spi, unsigned char* bitMap);
```

Comment:
Why are you initializing the LCD object with the bitmap? Wouldn't it make more sense to have a function like lcd.setBitMap()?

Reply:
It looks like displayMap() has the behavior I was describing. Why do we pass a bitmap here if we just pass another later?